leetcode/src/75.c

```diff
@@ -0,0 +1,47 @@
+#include <stdio.h>
+#include <string.h>
+#include <math.h>
+#include <stdlib.h>
+
+void sortColors(int* arr, int n){
+    
+    
+    void swap(int *x, int *y){
+    if (x==y)
+        return;
+   *x = *x + *y;
+    *y= *x - *y;
+    *x= *x - *y;
+}
+
+    int start=0, mid=0, end=n-1;
+    while(mid<=end){
+        if(arr[mid]==1)
+            mid++;
+        else if(arr[mid]==0){
+            swap(&arr[mid],&arr[start]);
+            mid++;
+            start++;
+        }
+        else{
+            swap(&arr[mid],&arr[end]);
+            end--;
+        }
+    }
+}
+
+
+int main(){
```

Comment:
Same here.

Reply:
I've updated the code and removed the header files that aren't required.